fhirpipe/errors.py

```diff
@@ -0,0 +1,2 @@
+class OperationOutcome(Exception):
```

Comment:
I would move this file to fhirpipe/api/errors.py

Reply:
Even if it's used in extract/graphql.py?

Reply:
mhh in that case not, you're right